tools/travis.ps1

```diff
@@ -154,8 +154,9 @@ else
 
 # Run a full build if the build was trigger via cron, api or the commit message contains `[Feature]`
 $hasFeatureTag = $commitMessage -match '\[feature\]'
+$hasPackageTag = $commitMessage -match '\[package\]'
```

Comment:
The line `Start-PSBootstrap -Package:(-not $isPr)` needs to be updated.
How about change to:
```powershell
$hasPackageTag = $commitMessage -match '\[package\]'
$createPackages = -not $isPr -or $hasPackageTag
...
Start-PSBootstrap -Package:$createPackages
...
if ($createPackages) {
   ## create package 
```

Reply:
will update